recipes/cli/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.15)
+project(PackageTest CXX)
+
+find_package(CLI CONFIG REQUIRED)
+
+add_executable(example example.cpp)
+target_link_libraries(example CLI::CLI)
```

For libraries that define a min cppstd greater than 11, we like to add this

```suggestion
target_link_libraries(example CLI::CLI)

target_compile_features(example PRIVATE cxx_std_14)
```